### PR TITLE
Landing page for LeagueSpec Profile

### PR DESCRIPTION
**Pull request by sl7v3**
This branch contains changes in the following areas:
- I added a header to all LeagueSpec pages in the _navigation.html.erb partial
- I created a skeleton for the LeagueSpec profile page. A lot of CSS still needs to be applied but I wanted to send something out.

------I'll wait a few days to see if Joel works on the API call, if not, I will start on it.

Review this branch and let me know any improvements needed.

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -81,5 +82,61 @@ input {
...
+}
+
+/* profile view */
+.profile_view { 
+  padding-top: 70px; 
+  width: 100%;
+  
+  .profile_info {
+  
+    }
```
Comment (danfstucky) [inline app/assets/stylesheets/application.css.scss]: You should remove the `.profile_info` selector if you don't have any styles specific to it.

**Comment by danfstucky** [comment]
The find friend button and search box in the navbar should be vertically aligned with one another.

**Comment by sl7v3** [comment]
Code in landing_page branch now has vertical alignment and css file cleaned up. Review and let me know.

**Comment by jwiebe17** [comment]
Sulai, what was that new environment you are using for ruby on rails.

**Comment by sl7v3** [comment]
Cloud9.io. You'll have to clone this repo when you sign up. If you already have a working Rails env though, you might just continue with that. I am using my initial Rails env for this LeagueSpec project.

**Comment by danfstucky** [comment]
+1

**Comment by jwiebe17** [comment]
Currently getting this error:
You have requested:                                                                                                                                                                                                                                  
 bcrypt ~> 3.1.10                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                     

The bundle currently has bcrypt locked at 3.1.11.                                                                                                                                                                                                    
Try running `bundle update bcrypt`                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                     
If you are updating multiple gems in your Gemfile at once,                                                                                                                                                                                           
try passing them all to `bundle update`                                                                                                                                                                                                              
Run `bundle install` to install missing gems.

**Comment by sl7v3** [comment]
Either change bcrypt in your Gemfile to 3.1.11 and run bundle install or just run bundle update bcrypt

**Comment by jwiebe17** [comment]
Neither of the error suggestions worked.

**Comment by danfstucky** [comment]
Joel, the error you are getting is unrelated to this pull request and I think will take some debugging.  Let's go ahead and merge this pull request for now and I will look into any bcrypt problems after.

**Comment by sl7v3** [comment]
Merged pull request

**Pull request merged**